utils/land-patterns.stanza

```diff
@@ -23,7 +24,75 @@ public defn ref-label (x:Double, y:Double) :
 public defn ref-label () :
   ref-label(0.0, 0.0)
 
-; ====== Land pattern utilities ===============================================
+;==============================================================================
+;======================== Geometry helpers ====================================
+;==============================================================================
+; Create a rectangle out of lines
+public defn LineRectangle (w:Double, h:Double, xc:Double, yc:Double, line-w:Double) :
+  val w2 = w * 0.5
```

Comment:
`half-w`?

Reply:
I think we could even do something here like

```
public defn LineRectangle (width:Double, height:Double, xc:Double, yc:Double, line-w:Double) :
  val p = points(to-polygon(Rectangle(width, height, loc(xc, yc))))
  Line(line-w, to-tuple $ cat-all([p, [p[0]]]))
```